.github/actions/deploy-environment/action.yml

```diff
@@ -35,3 +35,36 @@ runs:
             env:
               DOCKER_IMAGE_TAG: ${{ inputs.docker-image }}
               PR_NUMBER: ${{ inputs.pull-request-number }}
+
+          - name: Extract Terraform outputs
+            shell: bash
+            id: set_outputs
+            run: |
+              output=$(terraform -chdir=terraform/application output -json hostname)
+              echo "HOSTNAMES=$output" >> $GITHUB_ENV
+
+          - name: Set up Ruby 3.2.2
+            uses: ruby/setup-ruby@v1
+            with:
+              ruby-version: 3.2.2
+
+          - name: Bundle smoke test gems
+            shell: bash
+            run: |
+                gem install bundler
+                echo 'gem "rspec"' >> Gemfile
+                bundle
+
+          - name: Run RSpec tests
+            shell: bash
+            run: |
+             CLAIMS_EXTERNAL_HOST=$(echo $HOSTNAMES | jq -r '.[0]')
+             PLACEMENTS_EXTERNAL_HOST=$(echo $HOSTNAMES | jq -r '.[1]')
+
+             export CLAIMS_EXTERNAL_HOST=$CLAIMS_EXTERNAL_HOST
+             export PLACEMENTS_EXTERNAL_HOST=$PLACEMENTS_EXTERNAL_HOST
+
+             bin/bundle exec rspec --tag smoke_test -b
+            env:
+                SMOKE_TEST: "true"
+                HOSTNAMES: ${{ env.HOSTNAMES }}
```

Comment:
Might want to check if the devs want slack notifications for smoke test failures for review apps, as I think that will happen looking at the build-and-deploy workflow (you could test this out to be sure).

Reply:
We haven't enabled slack notification for the review App